app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    com, first_file, second_file = command.split()
+    if first_file != second_file:
+        with (open(first_file, "r") as file_in,
+              open(second_file, "w") as file_out):
+            data = file_in.read()
```

Comment:
You can skip creating a new variable `data` here and put .read() directly in the write()